### ISSUE TITLE
can't see updates

### ISSUE THREAD

**Issue by andreashandel** [epid8060fall2019/JuliaFrederick-project#1]
I can't see any of the part 2 components in the Rmd/doc file. Did you push the latest to Github? Am I looking in the wrong file? I looked in Project_Template.Rmd, it didn't contain anything related to cleaning and exploratory analysis.

**Comment by JuliaFrederick** [comment]
I pushed my updates from Part 2 on October 11. They included:
Incorporating text updates into the Project_Template.Rmd (mainly bibliography and explaining the dataset in more detail)
Cleaning/wrangling data to the processingcode
Also in the processingcode I included my preliminary analysis, because I wasn't sure if this is what I wanted to go after for my analysis code.  (I assumed the analysiscode document would be for a more finalized version of the analysis)

I think I'm misunderstanding where you say "main text" I was taking this to mean any of the main coding documents for some reason. So I was detailed in what I was doing throughout my cleaning code, but I'm guessing now that you actually want a word document or .Rmd file explaining this?

Overall the majority of my updates take place in the processingscript.R including the graphs and preliminary analysis. I think I'm just confused on what all of the files are for and when I should be using what. I can work on updating and separating out my code (and create a .Rmd file writing out what the cleaning code is doing). Especially because in Part of 1 of this project my lab hadn't finished assembling the data.

Let me know how you would like me to fix this. I think I have the pieces of what was asked they just aren't in the places expected due to a misunderstanding.
